### PR TITLE
Implement TramplesMinimize as a move flag

### PR DESCRIPTION
**Pull request by Keyacom**
Fixed PBS data updater so it does not update to nonexistent function codes.

### CODE REVIEW

**Comment by Maruno17** [comment]
_This is an accidental duplicate of another pull request, but with the above addition. The notes from that other pull request (which is now superseded by this one) are:_

Implemented `TramplesMinimize` as a move flag. Due to it, removed effect codes `FlinchTargetTrampleMinimize` and `ParalyzeTargetTrampleMinimize` (the second one was exclusive to Body Slam). Moves that had them now have the effect codes `FlinchTarget` and `ParalyzeTarget`, respectively.

The code now does not check if the parameter of `tramplesMinimize?` is 1 or 2, it now checks if it is true or false. For the effect that causes the moves with this flag to skip accuracy checks, it also checks if `Settings::MECHANICS_GENERATION` is equal to or greater than 6.

Data on which moves are to be treated as able to double damage if Minimized and skip accuracy checks are here.

Remarks:

- Body Slam, Dragon Rush, Heat Crash and Heavy Slam could not trample Minimize in Gen 5. Body Slam, Dragon Rush and Heat Crash could do so in Gen 6+, but Heavy Slam could not until Gen 7.
- Double Iron Bash could only trample Minimize in Gen 7.

TL;DR:

- Implemented `TramplesMinimize` as a move flag.
- Modified the check for this move flag.
- Removed effect codes `FlinchTargetTrampleMinimize` and `ParalyzeTargetTrampleMinimize`. Moves that had them now have the effect codes `FlinchTarget` and `ParalyzeTarget`, respectively.
